config/colorterms.py

```diff
@@ -1,52 +1,14 @@
 from lsst.pipe.tasks.colorterms import Colorterm, ColortermDict
 
 config.data = {
-    "hsc*": ColortermDict(data={
-        # Names used by Exposure.getFilter() in Gen2.
-        'g': Colorterm(primary="g", secondary="g"),
-        'r': Colorterm(primary="r", secondary="r"),
-        'i': Colorterm(primary="i", secondary="i"),
-        'z': Colorterm(primary="z", secondary="z"),
-        'y': Colorterm(primary="y", secondary="y"),
-        # Names used by data IDs in both Gen2 and Gen3, and
-        # Exposure.getFilter() in Gen3 (data is the same).
-        'HSC-G': Colorterm(primary="g", secondary="g"),
-        'HSC-R': Colorterm(primary="r", secondary="r"),
-        'HSC-I': Colorterm(primary="i", secondary="i"),
-        'HSC-Z': Colorterm(primary="z", secondary="z"),
-        'HSC-Y': Colorterm(primary="y", secondary="y"),
```

Comment:
I don't agree with removing the HSC -> HSC no-op option.  I think that for convenience it's nice to be able to calibrate an HSC image against an HSC reference catalog, and let it do the straight pass-through when you leave color terms "on".
Though we then have to add on all the physical filters (including NB) to this list.

Reply:
Ok, I've left these as-is. I'll leave it to someone else to expand them if desired.